### ISSUE TITLE
Make onChange handler optional

### ISSUE THREAD

**Issue by deepakjois** [chenglou/react-radio-group#17]
If I do no specify an onChange handler, for e.g.

```
 <RadioGroup ref="deliverymethod" name="deliverymethod">
...
</RadioGroup>
```

I get an error:

```
Uncaught TypeError: Cannot read property 'bind' of undefined
```

Does the onChangeHandler need to be set. Can’t the React element maintain its own state, and update it when the value changes.

**Comment by chenglou** [comment]
If you don't provide `onChange`, you'll receive the warning that the handler's not set, because the library allows only [controlled components](http://facebook.github.io/react/docs/forms.html#controlled-components) for now. Maybe we should implement uncontrolled components, but that gets slightly messy.

**Comment by briandamaged** [comment]
I just created a PR that allows both the "onChange" and "selectedValue" properties to be optional.  If neither property is specified, then React will treat the radio buttons as uncontrolled components.

**Comment by nkbt** [comment]
Fixed by #22
